rails_app/app/views/homepage/index.html.haml

```diff
@@ -1,6 +1,32 @@
 .well
-  .input-group
-    %input.form-control{type: 'text', placeholder: 'www.example.com'}
-    %span.input-group-btn
-      %button.btn.btn-default{type: 'button', type: 'button'}
-        Search!
+  - if current_user
+    = image_tag 'phone1.gif'
+    .header
+      %h1 Paster url to track!
```

Comment:
`Paste` if you will, but I think this whole `h1` is unnecessary

Reply:
h1 is styling, .header is wraping